tests/__fixtures__/cardanoSignTransaction.js

```diff
@@ -250,14 +264,19 @@ const SAMPLE_CERTIFICATES = {
     },
 };
 
-const SAMPLE_WITHDRAWAL = {
-    path: "m/1852'/1815'/0'/2/0",
-    amount: '1000',
-};
-
-const SAMPLE_WITHDRAWAL_SCRIPT = {
-    scriptHash: '29fb5fd4aa8cadd6705acc8263cee0fc62edca5ac38db593fec2f9fd',
-    amount: '1000',
+const SAMPLE_WITHDRAWALS = {
+    basic: {
+        path: "m/1852'/1815'/0'/2/0",
+        amount: '1000',
+    },
+    script: {
+        scriptHash: '29fb5fd4aa8cadd6705acc8263cee0fc62edca5ac38db593fec2f9fd',
+        amount: '1000',
+    },
+    key_hash: {
+        keyHash: '29fb5fd4aa8cadd6705acc8263cee0fc62edca5ac38db593fec2f9fd',
```

Comment:
Let's use a `keyHash` which is different from `scriptHash`. E.g. `3a7f09d3df4cf66a7399c2b05bfa234d5a29560c311fc5db4c490711` is used as a keyHash in FW tests. Please also update this in the docs certificates and withdrawals.

Reply:
Fixed.